server/config/passport.js

```diff
@@ -0,0 +1,32 @@
+var GitHubStrategy = require('passport-github').Strategy
+const store = require('store')
+
+module.exports = function (passport) {
+  passport.serializeUser(function (user, done) {
+    done(null, user.id)
+  })
+
+  // used to deserialize the user
+  passport.deserializeUser(function (id, done) {
+    const user = store.get('user')
+    done(null, user)
+  })
+  passport.use(new GitHubStrategy({
+    clientID: process.env.GITHUB_CLIENT_ID,
+    clientSecret: process.env.GITHUB_CLIENT_SECRET,
+    callbackURL: 'https://smee.io/auth/github/callback'
+  },
+  function (token, tokenSecret, profile, done) {
+    process.nextTick(function () {
+      const user = {
+        id: profile.id,
+        displayName: profile.displayName,
+        username: profile.username,
+        profilePic: profile._json.avatar_url,
+        token: token
+      }
+      store.set('user', user)
+      return done(null, user)
+    })
+  }))
```

Comment:
`Store` is a cross browser storage for all use cases. It will pick the best available storage, and automatically falls back to the first available storage that works like
****
****
****